LICENSE

```diff
@@ -0,0 +1,23 @@
+The original contents of the nodejs.org repo are licensed for use as follows:
+
+"""
```

Comment:
Curious what these triple double quotes are.

There's a corresponding set on line 23.

Reply:
As I mentioned, I just copy/pasted the same license as the Nodejs.org repo.

Reply:
Having a license is a vital step 👍